src/features/classification/security-controls/SecurityControls.js

```diff
@@ -80,14 +80,16 @@ class SecurityControls extends React.Component<Props, State> {
         const { isSecurityControlsModalOpen } = this.state;
         const shouldShowSecurityControlsModal =
             controlsFormat === SHORT_WITH_BTN && !!itemName && !!classificationName && !!definition;
+        const securityControlsItems = items.map(({ message, tooltipMessage }) => {
+            if (message) {
```

Comment:
I think we could avoid this extra check and the `return` below with the approach i mentioned above

Reply:
nice !